### ISSUE TITLE
Missing anchor points on Mandaic contextual forms

### ISSUE THREAD

**Issue by dscorbett** [notofonts/mandaic#1]
### Font
NotoSansMandaic-Regular.ttf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/af306de71ce51b9ce75389b163026427953606cf/phaseIII_only/unhinted/ttf/NotoSansMandaic/NotoSansMandaic-Regular.ttf
Date: 2019-04-25

### Font version
Version 2.000

### Issue
Contextually wider or narrower forms of Mandaic letters are missing anchor points.

### Character data
ࡂࡈ࡙
U+0842 MANDAIC LETTER AG
U+0848 MANDAIC LETTER ATT
U+0859 MANDAIC AFFRICATION MARK

### Screenshot
![ࡂࡈ࡙](https://user-images.githubusercontent.com/1124347/57180456-a8381000-6e56-11e9-9d62-a414610605b5.png)
With 'calt' disabled:
![ࡂࡈ࡙](https://user-images.githubusercontent.com/1124347/57180457-a8381000-6e56-11e9-9fc8-172959a22fed.png)

**Comment by LornaSIL** [comment]
Is there any chance this will get fixed? This seems to be the only Mandaic font available and without these two characters working, it's really unusable.
Or perhaps my problem is different. The two characters above look fine in Mandaic WF

Go here: https://r12a.github.io/pickers/mandaic/
Copy this text: ࡂࡁࡂࡃࡄࡉࡈࡇࡆࡒࡑࡋ
And it looks fine:
![image](https://user-images.githubusercontent.com/7071907/77959482-436e7b00-729c-11ea-86d6-e349a576c4de.png)
The font says it is Madaic WF. Change the font to Noto Sans Mandaic and it looks like this:
![image](https://user-images.githubusercontent.com/7071907/77959682-9cd6aa00-729c-11ea-93fd-281621bba3d9.png)
I have tried the same text on Word in Win10, Notepad and various other applications, using Noto Sans Mandaic and those two characters always shift down.

**Comment by dscorbett** [comment]
To be clear, this bug report is about anchor points. The two letters ag and aṭ are just an example. You have found a different problem.

**Comment by punchcutter** [comment]
There are actually anchors on these glyphs, but the fontmake pipeline expects the script properties to also be set on the glyphs. That's a very easy fix. 

What @LornaSIL shows is something else, but I'm not sure what yet.

**Comment by LornaSIL** [comment]
Do I need to create a separate issue?

**Comment by punchcutter** [comment]
Yes, that's a different issue which so far I can't reproduce on my Win10.

**Comment by LornaSIL** [comment]
Ok, I installed the font linked to above and it doesn't have the problem I was seeing. The font I was using is also labeled as ver 2.000, but the version string is quite different. Here are the properties:
![image](https://user-images.githubusercontent.com/7071907/77964547-d7dcdb80-72a4-11ea-88e8-3923e57b4ed8.png)

So, the version linked to above seems to have resolved the issue I was seeing. I downloaded the font I was using from here: https://www.google.com/get/noto/

I'm sorry for the diversion.
